lib/manageiq/appliance_console/cli.rb

```diff
@@ -232,6 +249,23 @@ def set_external_db
       config.post_activation
     end
 
+    def replication
+      if options[:replication] == "primary"
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationPrimary.new
+        say("Configuring Server as Primary")
+      else
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationStandby.new
+        say("Configuring Server as Standby")
+        db_replication.disk = disk_from_string(options[:dbdisk])
+        db_replication.primary_host = options[:primary_host]
+        db_replication.standby_host = options[:standby_host]
```

Comment:
This will default to the current appliance's address. So we can make this parameter optional and only set the attribute on `db_replication` if it was provided.